src/api/flaskr/service/metering/__init__.py

```
@@ -0,0 +1,8 @@
+from flaskr.service.common.dicts import register_dict
+
+from .consts import BILL_USAGE_SCENE_DICT, BILL_USAGE_TYPE_DICT
+from .models import BillUsageRecord  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)